website/docs/r/advanced_cluster.html.markdown

```diff
@@ -246,6 +246,7 @@ replication_specs {
 
 * `num_shards` - (Required) Provide this value if you set a `cluster_type` of SHARDED or GEOSHARDED. Omit this value if you selected a `cluster_type` of REPLICASET. This API resource accepts 1 through 50, inclusive. This parameter defaults to 1. If you specify a `num_shards` value of 1 and a `cluster_type` of SHARDED, Atlas deploys a single-shard [sharded cluster](https://docs.atlas.mongodb.com/reference/glossary/#std-term-sharded-cluster). Don't create a sharded cluster with a single shard for production environments. Single-shard sharded clusters don't provide the same benefits as multi-shard configurations.
 * `region_configs` - (Optional) Configuration for the hardware specifications for nodes set for a given regionEach `region_configs` object describes the region's priority in elections and the number and type of MongoDB nodes that Atlas deploys to the region. Each `region_configs` object must have either an `analytics_specs` object, `electable_specs` object, or `read_only_specs` object. See [below](#region_configs)
+* `region_configs_container_ids` - A key-value map for Network Peering Container ID with syntax as `"providerName:regionName" = "containerId"`. Example `AWS:US_EAST_1" = "61e0797dde08fb498ca11a71`.
```

Comment:
Why not call this container_id like we did with clusters (container_ids is fine too but really just keeping the same should be fine)?  It can still be a map but just explained via this doc, as you've done.  Something like: 

```suggestion
* `container_id` - A key-value map of the Network Peering Container ID. The Container ID is the id of the container either created programmatically by the user before any clusters existed in a project or when the first cluster in the region (AWS/Azure) or project (GCP) was created.  The syntax is `"providerName:regionName" = "containerId"`. Example `AWS:US_EAST_1" = "61e0797dde08fb498ca11a71`.

```

Reply:
Because is not inside of regions_config, and is a type map because it can appear more containers ids if there's more region configs in the config file, and it will appear like this 
```
"AWS:US_EAST_1" = "61e0797dde08fb498ca11a71",
"AWS:US_EAST_2" = "61e0797dde08fb498ca11a72",
```
For some reason if I put the computed parameter `container_id` in the `regions_config` it will always appear a diff after plan even if you don't make changes, not sure why happens, maybe it's because is a TypeSet of another TypeSet, but after reading this [solution](https://discuss.hashicorp.com/t/custom-provider-how-to-reference-computed-attribute-of-typemap-list-set-defined-as-nested-block/22898/2) and I applied the same, and now it works without appearing a diff after plan.

Reply:
Okay I get why you would name it that because of how it works but that's still likely better handled in the description of the documentation imo, e.g.:

 `container_id` - A key-value map of the Network Peering Container ID(s) for the configuration specified in `region_configs`. The Container ID is the id of the container either created programmatically by the user before any clusters existed in a project or when the first cluster in the region (AWS/Azure) or project (GCP) was created.  The syntax is `"providerName:regionName" = "containerId"`. Example `AWS:US_EAST_1" = "61e0797dde08fb498ca11a71`.

Reply:
Understood, will make changes, thank you.